storybook/src/stories/person/card-example.stories.tsx

```diff
@@ -86,33 +87,34 @@ export const Size: Story<{ sizes: Array<CardProps['size']> }> = (props: { sizes:
 Size.args = { sizes: ['small', 'medium', 'large'] };
 
 export const DataSource: Story<CardProps> = (props: CardProps) => (
-  <PersonProvider resolve={createResolve()}>
-    <PersonCard {...props} dataSource={{
-              name: 'Rick Sanchez (C-137)',
-              pictureSrc: 'https://i.imgur.com/17Kw9my.jpg',
-              accountType: PersonAccountType.Enterprise,
-              jobTitle: 'Scientist',
-              mail: 'rick.sanchez@email.com',
-              mobilePhone: '+47 123456789',
-              manager: {
-                  azureUniqueId: '1234-1324-1235',
-                  name: 'Justin Roiland',
-                  department: 'Adult Swim',
-                  pictureSrc:
-                  'https://i.imgur.com/2TO7k63.jpeg',
-                  accountType: PersonAccountType.Employee,
-              },
-              positions: [
-                  {
-                      id: '123-123',
-                      name: 'Wubba Lubba Dub Dub',
-                      project: {
-                          id: '1234-1234',
-                          name: 'The Citadel',
-                      },
-                  },
-              ],
-          }} />
+  <PersonProvider personResolver={createResolve}>
+    <PersonCard {...props} azureId='9876-5432-1098' dataSource={{
```

Comment:
azure can be omitted when dataSource is defined.

Reply:
nope, not working like that any more, can't resolve image without azureId (for now)